db/sql/create-timeEntry-and-timePeriodType-table.sql

```diff
@@ -0,0 +1,32 @@
+
+CREATE TABLE timecard.time_period_type (
+    id VARCHAR(36)  PRIMARY KEY,
+    tenant_id VARCHAR(36) NOT NULL,
+    name VARCHAR(50) NOT NULL
+);
+
+INSERT INTO timecard.time_period_type (id,tenant_id,"name") VALUES
+	 ('00000000-0000-0000-0000-000000000001','00000000-0000-0000-0000-000000000000','Shift'),
+	 ('00000000-0000-0000-0000-000000000002','00000000-0000-0000-0000-000000000000','Scheduled rest day'),
+	 ('00000000-0000-0000-0000-000000000003','00000000-0000-0000-0000-000000000000','Non-working day'),
+	 ('00000000-0000-0000-0000-000000000004','00000000-0000-0000-0000-000000000000','On call'),
+	 ('00000000-0000-0000-0000-000000000005','00000000-0000-0000-0000-000000000000','Absence'),
+	 ('00000000-0000-0000-0000-000000000006','00000000-0000-0000-0000-000000000000','Training'),
+	 ('00000000-0000-0000-0000-000000000007','00000000-0000-0000-0000-000000000000','Overtime');
+
+CREATE TABLE timecard.time_entry (
+    id VARCHAR(36)  PRIMARY KEY,
+    tenant_id VARCHAR(36) NOT NULL,
+    version INT DEFAULT NULL,
+    owner_id INT NOT null,
+    time_period_type_id VARCHAR(36) NOT NULL,
+    shift_type VARCHAR(15) DEFAULT NULL,
+    actual_start_time TIMESTAMP NOT NULL,
+    actual_end_time TIMESTAMP DEFAULT NULL,
+    planned_start_time TIMESTAMP DEFAULT NULL,
+    planned_end_time TIMESTAMP DEFAULT NULL,
+    created_at TIMESTAMP NOT NULL,
```

Comment:
do we need a created_by, updated_by, deleted_by?

Reply:
Sorry i got misunderstand here, This question should be clarified by David/Will

Reply:
As discussed briefly in standup this morning, I think the prevailing opinion is that we don’t want to be including database fields that’ll only be useful for features that we haven’t build yet, so I would vote for us not including created_by etc at this stage.

Going to resolve the conversation but happy to be challenged on this 🙇